src/streamlink_cli/output.py

```diff
@@ -99,6 +101,20 @@ def running(self):
         sleep(0.5)
         return self.player.poll() is None
 
+    @classmethod
+    def supported_player(cls, cmd):
```

Comment:
I think this method still needs a bit of work. Need to write some extensive tests for it :)

Reply:
I think this review is outdated now. I saw tests for the support_player method in your PR.

Reply:
Yep it is.